File: tasks/1.py
# 1. Two Sum

class Solution:
    def twoSum(self, nums: list[int], target: int) -> list[int]:
        for i, value in enumerate(nums):
            for j in range(i + 1, len(nums)):
                if value + nums[j] == target:
                    return [i, j]

File: solutions/1/1.go
package _1

import (
	"slices"
)

func twoSum(nums []int, target int) []int {
	for i, v := range nums {
		idx := slices.Index(nums[i+1:], target-v)

		if idx != -1 {
			return []int{i, i + idx + 1}
		}
	}

	return []int{}
}
